scripts/testnet/sovereignBridge/prerequisites.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+runBridgePrerequisites() {
```

Comment:
Where is this called and used?

Reply:
It's in the docs.
Step 2 is clone `mx-chain-go` and run `./prerequisites.sh` to install node dependencies.
Step 3 is `runBridgePrerequisites` to install all software needed for scripts and download the contracts automatically.